config/config-diversity-toolkit/.gitignore

```diff
@@ -1147,13 +1147,10 @@ views.view.users_by_group.yml
 system.action.user_add_role_action.messaging_manager.yml
 system.action.user_remove_role_action.messaging_manager.yml
 user.role.messaging_manager.yml
-field.storage.node.field_short_text.yml
```

Comment:
@melwong-jcc This line has removed from diversity toolkit. Can you please make sure.